app/services/configuration/energy_costs_default.py

```diff
@@ -0,0 +1,307 @@
+"""
+Configuration functions including default values for
+energy plans, usage profiles, and answers.
+
+Rewiring Aotearoa:
+    • Electricity variable cost: 24.2c/kWh or 18c/kWh off-peak/ripple.
+    • Electricity connection cost: Not included in any calculations.
+        (Homes are assumed on-grid in all cases.)
+    • Piped gas variable cost: 11c/kWh.
+    • Piped gas connection cost: $1.60 per day ($587 per year).
+    • LPG variable cost: 24.4c/kWh.
+    • LPG bottle rental (2 x 45kg bottles):
+        $5.75 per bottle per month ($138 per year for two bottles).
+
+Petrol and Diesel pricing:
+
+From MBIE: Average Petrol and Diesel Costs from 1/09/2023 to 23/08/2024
+    • Diesel_discounted_retail_price_NZc.p.l = 216.1612458
+    • Regular_Petrol_discounted_retail_price_NZc.p.l = 278.716297
+Source:
+    https://www.mbie.govt.nz/building-and-energy/
+energy-and-natural-resources/energy-statistics-and-modelling/
+energy-statistics/oil-statistics
+
+Wood pricing:
+
+Base this on:
+    • Price per cord: NZD $375
+    • Volume of a Cord = 128 cubic feet = 3.62m^3
+    • Density of dry pine = 480 kg/m^3
+    • Energy content of dry pine = 15 MJ / tonne
+        = 15E6 / 3.6 / 1E6 = 4.17 kWh / kg
+
+Exclude the following factor:
+    • Efficiency of modern wood stove = 70%
+We calculate the price per kWh of heat content in the wood and
+account for the efficiency of a modern wood stove separately.
+
+kWh per dollar = (
+    Volume of a Cord *
+    Density of dry pine *
+    Energy content of dry pine *
+    Efficiency of modern wood stove) / Price per cord
+ = (3.62 * 480 * 4.17) / 375
+ = 19.32 kWh per dollar
+
+Inverting this gives $0.052 per kWh of heat in the wood.
+
+This works out to $0.074 per kWh of heat output from a modern wood stove.
+"""
```

Comment:
For wood pricing, I got the below prices based on the Consumer NZ study and worked on the wood price using a similar approach. The values are pretty close.  [Buying firewood: how to get a hot deal - Consumer NZ](https://www.consumer.org.nz/articles/firewood)

Price per m³ (cubic meter) of each type of wood:
 
Pine (softwood): NZD $120/m³
Gum (hardwood): NZD $160/m³
Macrocarpa (medium-density wood): NZD $152/m³
Hot mix: NZD $140/m³
Densities and energy content for each type of wood:
 
Density of pine (softwood): 480 kg/m³
Density of gum (hardwood): 700 kg/m³
Density of macrocarpa (medium-density wood): 540 kg/m³
Density of hot mix: 560 kg/m³
Energy content of dry wood: 4.17 kWh/kg
 
Exclude the following factor:
Efficiency of modern wood stove = 70%

Pine (softwood): $0.086 per kWh
Gum (hardwood): $0.078 per kWh
Macrocarpa (medium-density wood): $0.096 per kWh
Hot mix: $0.086 per kWh

Reply:
Thanks a lot Achini. Taking a (weighted?) average of those numbers seems like an improvement on my calculation which was based only on pine. If so I'll update the comments to document the new approach. Happy with this approach @gg-eeca, @EECASmartv ?

Reply:
@Achieeca do you have a spreadsheet with calcs and references? I've got similar numbers from some work I did a few months ago in response to an EECA social media enquiry on wood stuff. I used UK and EU references, and have some calcs on the "Wood" tab of: https://eecagovtnz.sharepoint.com/:x:/s/MarketingandCommunications/ERo9wA79jclMmkyxz7kNYCIB6TEY9kMQ8N17Ps4WAsPWrQ?e=9X2exG

Reply:
Actually, double checking the wood numbers provided above, these look to me to be too big by a factor of 1/0.7 because they account for the 70% wood burner efficiency. What we are after (in order to fall in with the treatment of other fuels) is a cost in $/kWh carried by the fuel.